Makefile

```diff
@@ -0,0 +1,30 @@
+.PHONY: all clean format build test dev install-pre-commit public-header
+
+all: build test
+
+clean:
+	rm -rf build
+	rm -rf third_party/builds
+
+build:
+	earthly +build
+
+test:
+	earthly +test
+
+format:
+	git ls-files '**/*.c' '**/*.h' | grep -v -E '/mock_.*\.[ch]$$' | xargs clang-format -i -style=file
+
+# Install all tools required for development
+dev: install-pre-commit
```

Comment:
Also it is better to add it to README or somewhere ?

Reply:
Done.